### ISSUE TITLE
Enhancement: Implement ignore list

### ISSUE THREAD

**Issue by bfelder** [SublimeText/WordHighlight#73]
I like to be able to specify a list of words to be ignored.
This would improve the user experience.

For some frequently occurring words highlighting does not make sense and can even be confusing.
Consider a long ToDo list from PlainTasks. 
Here the check box symbol (☐) is repeated many times. Touching it with the caret leads to a whole column in the active view to be highlighted, yet no additional insight is provided to the user as the respective symbol is obviously positioned.

**Comment by matthewjmink** [comment]
I would add that a file extension ignore list would be helpful. For the most part, highlighting words in HTML files is not helpful for me. I would like to be able to exclude HTML files in the plugin settings. Another option would be pattern matching, so that HTML tags, for example, could be excluded.

**Comment by fitzliputz** [comment]
I agree to both previous suggestions. The package os otherwise fine, with the mentioned ignore options it would be phantastic. Thanks.
